src/components/InputButton/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Input, Button } from '@ht6/react-ui';
+import { ReactNode } from 'react';
+import cx from 'classnames';
+import { container, field, button } from './InputButton.module.scss';
+
+export interface InputButtonProps {
+  className?: string;
+  label: string;
+  name: string;
+  status?: {
+    state: "success" | "error";
+    text?: ReactNode;
+  };
+  onSubmit: (event: any) => any;
+  onChange: (event: any) => any;
+}
+
+function InputButton({
+  className,
+  label,
+  name,
+  status,
```

Comment:
Let's pull the type from the InputLabel itself (etc. `InputLabelProps['label']`) to have a single source of truth

Reply:
You mean `InputLayoutProps` right? Fixed.

Reply:
yes :p my bad